dashboard/cron.py

```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
 import logging
 from collections import namedtuple
+from operator import truediv
```

Comment:
It doesn't look like this truediv is being used.

Reply:
also reported by Codacy. This line is removed.